ansible.cfg

```diff
@@ -5,8 +5,9 @@ display_skipped_hosts = false
 verbosity = 1
 nocows = 1
 inventory = $PWD/inventories/
-roles_path = ./playbooks/galaxy_roles:./user_playbooks/roles:./roles
+roles_path = ./playbooks/galaxy_roles:./user_playbooks/roles:./roles:./ansible-pulp/roles
```

Comment:
I don't like this ansible-pulp specific thing. Could we generalize this somehow to accept any roles? I can also see a use case for foreman-ansible-modules. Ideally it could pull anything from galaxy.

Reply:
@ehelms thoughts?